fusequery/query/src/sessions/sessions.rs

```diff
@@ -60,6 +60,10 @@ impl SessionManager {
         }))
     }
 
+    pub fn get_conf(&self) -> &Config {
```

Comment:
If we want to return ref of Config, maybe get_conf -> `get_conf_ref` is better?

Reply:
I am not sure... 

Since there are `DataSchemaRef`, `SessionManagerRef` ..., it seems that if `ref` is explicitly mentioned, it means `Arc<Something>`.

But I am Ok with it, if `get_conf_ref` is preferred, I'd like to rename it.

Reply:
Bad name ideas from arrow, type ArrayRef = Arc<dyn Array>;